misc/services/acrn_manager/README.rst

```diff
@@ -87,7 +87,7 @@ Use the ``list`` command to display VMs and their state:
 Start VM
 ========
 
-If a VM is in a ``stopped`` state, you can start it with the ``start``
+If a VM is in a ``stopped`` state, you can start it with the ``start``, acrnd automatically loads launch scripts under /usr/share/acrn/conf/add/ to boot VMs.
```

Comment:
Please make these edits to lines 90-91:

If a VM is in a ``stopped`` state, you can start it with the ``start`` command. acrnd automatically loads launch scripts under ``/usr/share/acrn/conf/add/`` to boot the VMs.